react-frontend/src/shared-components/core-value-icon-components/CoreValueIconComponent.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { BsCircle } from "react-icons/bs";
```

Comment:
create an icon directory in core-components and a file icons.js export needed icons from there, just like we added abstraction for react-bootstrap